Block/SegmentMapping.php

```diff
@@ -86,10 +87,11 @@ public function getNostoAccount()
         try {
             $store = $this->storeManager->getStore();
         } catch (\Exception $e) {
-            $this->logger->exception("Could not get Nosto account ID");
+            $this->logger->error($e->getMessage() . 'Could not get Nosto account ID');
```

Comment:
Why was this changed?

Reply:
Exception receives an object `\Exception`, error receives a message `String`

Reply:
Ah, my bad!